Makefile

```diff
@@ -25,8 +25,14 @@ dev: node_modules/.uptodate python
 	@tox -qe dev
 
 .PHONY: lint
-lint: node_modules/.uptodate python
+lint: backend-lint frontend-lint
+
+.PHONY: backend-lint
+backend-lint: python
 	@tox -qe lint
+
+.PHONY: frontend-lint
+frontend-lint: node_modules/.uptodate
 	@./node_modules/.bin/eslint bouncer/scripts
```

Comment:
Looks like we never got around to setting up Prettier for front-end code formatting in this repo. Oh well, exercise for another time.